doc/nrf/ug_thread_architectures.rst

```diff
@@ -30,29 +33,31 @@ This can be done per platform, while retaining the ability to default to a stand
 System-on-Chip designs
 **********************
 
-This single-chip solution has the combined RFIC (the IEEE 802.15.4 in case of Thread) and processor.
-OpenThread and the application layer run on the local processor.
+A single-chip solution uses the RFIC (the IEEE 802.15.4 in case of Thread) and the processor of a single SoC.
+
+In these designs, OpenThread and the application layer run on the same local processor.
 
 .. _thread_architectures_designs_soc_designs_single:
 
 Single-chip, single protocol (SoC)
 ==================================
 
-In this design, the application layer and OpenThread run on the same processor.
+In this design, the only wireless protocol that is used is Thread.
+OpenThread and the application layer run on the same processor.
 The application uses the OpenThread APIs and IPv6 stack directly.
 
-This is the SoC design most commonly used for applications that do not make heavy computations or are battery-powered.
+This SoC design is most commonly used for applications that do not make heavy computations or are battery-powered.
 
 This design has the following advantages:
 
-* Lowest cost.
-* Lowest power consumption.
-* Lowest complexity.
+* It requires only one chip, resulting in a low cost.
```

Comment:
I'm not sure if this is precise enough. There is one chip that combines the radio and general MCU functionalities.
In most applications we see more chips, e.g. sensors, memories, front end modules and so on.